learn/configuration/synonyms.md

```diff
@@ -122,4 +124,4 @@ Suppose you set `San Francisco` and `SF` as synonyms with a [mutual association]
 }
 ```
 
-If you input `SF` as a search query, then results containing `San Francisco` will also be returned. However, **they will be considered less [relevant](/learn/core_concepts/relevancy.md) than those containing `SF`**. The reverse is also true.
+If you input `SF` as a search query, Meilisearch will also return results containing the phrase `San Francisco`. However, **they will be considered less [relevant](/learn/core_concepts/relevancy.md) than those containing `SF`**. The reverse is also true: if your query is `San Francisco`, documents containing `San Francisco` will rank higher than those containing `SF`.
```

Comment:
Nitpicking: they are not always considered less relevant, it depends on the ranking rules that are chosen. I think (60% sure) they are only considered less relevant if the `exactness` ranking rule is used. Maybe this is not a detail worth mentioning in the documentation. Also I think we _might_, in the future, actually make them always less relevant, but it's difficult to implement.